components/api_version/src/api_v2.rs

```diff
@@ -168,6 +169,42 @@ impl KvFormat for ApiV2 {
             encoded_key
         }
     }
+
+    // add prefix RAW_KEY_PREFIX
+    fn convert_encoded_key_version_from(
+        src_api: ApiVersion,
+        key: &[u8],
+        ts: Option<TimeStamp>,
+    ) -> Result<Key> {
+        match src_api {
+            ApiVersion::V1 | ApiVersion::V1ttl => {
+                let mut apiv2_key = Vec::with_capacity(ApiV2::get_encode_len(key.len() + 1));
+                apiv2_key.push(RAW_KEY_PREFIX);
```

Comment:
This might confuse the readers since the name of the function declares it can convert data from versions including txnkv `src_api` to ApiV2 or other versions, but it adds a raw prefix to the key instead of a txn prefix. Suggesting leave a TODO or rename the function.

Reply:
good suggestion. I have renamed the function names, raw is added. PTAL~